content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -95,42 +86,52 @@ There may be cases when a workspace contains conflicting variables of the same t
 
 Terraform Cloud prioritizes and overwrites conflicting variables according to the following precedence:
 
-### 1. Workspace-Specific Variables
+### 1. Command line Argument Variables
+
+When using a CLI workflow, variables applied to a run with either `-var` or `-var-file` overwrite workspace-specific and variable set variables that have the same key.
+
+### 2. Local Environment Variables Prefixed with `TF_VAR_`
+
+When using a CLI workflow, local environment variables prefixed with `TF_VAR_` (e.g., `TF_VAR_key`) overwrite workspace-specific, variable set, and `.auto.tfvars` file variables that have the same key.
```

Comment:
i read it again and realized key might be confusing because we keep talking about variables with the same key elsewhere. 

```suggestion
When using a CLI workflow, local environment variables prefixed with `TF_VAR_` (e.g., `TF_VAR_replicas`) overwrite workspace-specific, variable set, and `.auto.tfvars` file variables that have the same key.
```